prometheus-collectors/class-potential-multi-dataset-queries-collector.php

```diff
@@ -49,6 +50,21 @@ public function sql_query_log( $query ): void {
 					$blog_ids_count >= 3 ? '3+' : (string) $blog_ids_count,
 				]
 			);
+
+			$backtrace = function_exists( 'wp_debug_backtrace_summary' ) ? wp_debug_backtrace_summary( null, 4, false ) : []; // phpcs:ignore
+			\Automattic\VIP\Logstash\log2logstash(
```

Comment:
I just realized that the logstash code is loaded during mu-plugins but we can have queries run before that which will cause a fatal here.

We should make sure that the function exists before using it (I can't remember how that works with namespaces, and if we just need it here or if the `use` above is at risk as well).

(I guess this is where having an autoloader for library functions like this would be nice :( )

Reply:
Since this collector is also part of the mu-plugins, and the files (`prometheus.php` and `logstash.php`) are required at `000-vip-init.php`, I think everything is going to be loaded properly when we call `Potential_Multi_Dataset_Queries_Collector->initialize() -> add_action( 'sql_query_log' ...)`.

Just to make it risk-free, I added the check you suggested. 

What do you think?